lib/geordi/util.rb

```diff
@@ -60,19 +61,23 @@ def run!(command, show_cmd: false, confirm: false, fail_message: 'Something went
 
         if testing?
           # Join with commas for precise argument distinction
-          puts "Util.run! #{show_command.join(', ')}"
+          puts "Util.run!#{' (exec)' if exec} #{show_command.join(', ')}"
         else
+          method = exec ? :exec : :system
+
           # Remove Geordi's Bundler environment when running commands.
           success = if !defined?(Bundler)
-            system(*command)
+            Kernel.public_send(method, *command)
           elsif Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('1.17.3')
             Bundler.with_original_env do
-              system(*command)
+              Kernel.public_send(method, *command)
             end
           else
-            Bundler.clean_system(*command)
+            method = exec ? :clean_exec : :clean_system
+            Bundler.public_send(method, *command)
           end
 
+          # This part will never be reached when `exec` is true
```

Comment:
Had to google once to find out how `exec` works. 

For my understanding: This part is never reached because Kernel#exec replaces the current process (geordi) with the process called by exec (rails console)?

Reply:
That's right, see https://makandracards.com/makandra/1243-execution-of-shell-code-in-ruby-scripts#section-exec-show-archive-org-snapshot-the-last-thing-you-do